tensorflow/compiler/xla/service/gpu/gpu_conv_algorithm_picker.cc

```diff
@@ -659,7 +663,7 @@ GpuConvAlgorithmPicker::PickBestAlgorithmNoCacheCuda(
   se::RedzoneAllocator input_output_allocator(
       stream, allocator,
       PtxOptsFromDebugOptions(hlo_module_config.debug_options()),
-      /*memory_limit=*/se::RedzoneAllocator::kDefaultMemoryLimit,
```

Comment:
Is kDefaultMemoryLimit constant unused then? Can it be removed?

Reply:
I found the redzone_allocator_test.cc still uses the constant. Should I remove them and use the literals instead (e.g. 1LL<<32)?

Reply:
yes, test-only usage should be discouraged.

Reply:
Sure. Just pushed a new commit. PTAL.